network/peer/peer.go

```diff
@@ -948,6 +956,17 @@ func (p *peer) handleHandshake(msg *p2p.Handshake) {
 	}
 
 	// handle subnet IDs
+	if numTrackedSubnets := len(msg.TrackedSubnets); numTrackedSubnets > maxNumTrackedSubnets {
+		p.Log.Debug("message with invalid field",
```

Comment:
I intentionally used `"message with invalid field"` so that we could just search that in logs and see all the related logs. But I think there are a few inconsistencies here. I'll put up a PR to clean this up a tad.

Reply:
Added PR here: https://github.com/ava-labs/avalanchego/pull/3017